app/src/main/java/com/codesoom/models/TaskService.java

```diff
@@ -0,0 +1,92 @@
+package com.codesoom.models;
+
+import com.codesoom.HttpEnum.HttpStatusCode;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TaskService {
+
+    private ObjectMapper mapper = new ObjectMapper();
+    private Map<Long, Task> tasks = new HashMap<>();
+    private Long autoId = 1L; // 아이디를 자동으로 부여하기 위한 변수
+
+    public SendResponseData getAll() throws IOException {
+
+        if(!tasks.isEmpty()) {
+            return new SendResponseData(HttpStatusCode.OK.getStatus(),toTaskJson());
+        } else {
+            return new SendResponseData(HttpStatusCode.OK.getStatus(),"[]");
+        }
```

Comment:
사실 여기에서는 `else`가 필요하지 않습니다.
```suggestion
        if(!tasks.isEmpty()) {
            return new SendResponseData(HttpStatusCode.OK.getStatus(),toTaskJson());
        }
        return new SendResponseData(HttpStatusCode.OK.getStatus(),"[]");
```

Reply:
수정하였습니다 !